src/Popper/getReferenceElement.js

```diff
@@ -0,0 +1,18 @@
+const getReferenceElement = reference => {
+    // Elements or virtualElements
```

Comment:
Maybe a comment linking to: https://popper.js.org/docs/v2/virtual-elements would be nice here. I had no idea that this was an officially supported usecase and the term virtual element was new to me in this context.

Reply:
Good idea, I'll update the jsDocs